tests/jobs/correlated_observations_scaling/unit/test_scaling_job.py

```diff
@@ -64,12 +65,14 @@ def test_valid_job(calc_key, app_key, obs_keys, obs_with_data, expect_valid, err
         "CALCULATE_KEYS": {"keys": [{"key": calc_key}]},
         "UPDATE_KEYS": {"keys": [{"key": app_key}]},
     }
+
+    reporter = FileReporter(os.path.realpath(os.getcwd()))
```

Comment:
Would be nice with a test loading the published data again...

Reply:
I agree. I have moved it to the integration test though, as the scale functions are not run in the unit test